cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -189,3 +191,39 @@ def _cleanup():
     results = appliance.rest_api.response.json()["results"]
     assert results[0]["success"] is False
     assert results[1]["success"] is True
+
+
+@pytest.mark.parametrize("transformation_method", ["SSH", "VDDK"], ids=["SSH", "VDDK"])
+def test_rest_conversion_host_crud(appliance, source_provider, provider, transformation_method):
+    """
+    Tests conversion host crud via REST
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: high
+        testtype: functional
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: V2V
+    """
+    # Test1: Create two conversion host
+    set_conversion_host_api(appliance, transformation_method, source_provider, provider)
+    conv_host = appliance.rest_api.collections.conversion_hosts.all
+
+    # Test2: Edit first conversion host
+    fixed_limit = 100
+    edited_conv_host = appliance.rest_api.collections.conversion_hosts.action.edit(
+        id=conv_host[0].id,
+        max_concurrent_tasks=fixed_limit)[0]
+    assert edited_conv_host.max_concurrent_tasks == fixed_limit
+
+    # Test3: Delete both conversion host
+    for c in conv_host:
+        response = appliance.rest_api.collections.conversion_hosts.action.delete(id=c.id)[0]
+        wait_for(
+            lambda: response.state == "Finished",
+            fail_func=response.reload,
+            num_sec=240,
+            delay=3,
+            message="Waiting for conversion host configuration task to be deleted")
+    assert not bool(appliance.rest_api.collections.conversion_hosts.all)
```

Comment:
I don't think you need to convert the list to bool. Simply doing `assert not appliance.rest_api.collections.conversion_hosts.all` would do. If the list is empty, the assertion would pass.